### ISSUE TITLE
Not compatible with @svgdotjs/svg.js package

### ISSUE THREAD

**Issue by phaserwork** [svgdotjs/svg.resize.js#56]
I use @svgdotjs/svg.js package
"dependencies": {
    "@svgdotjs/svg.draggable.js": "^3.0.2",
    "@svgdotjs/svg.js": "^3.0.12",
    "core-js": "^2.6.5",
    "normalize.css": "^8.0.1",
    "vue": "^2.6.10",
    "vue-router": "^3.0.3"
  }

**Comment by Fuzzyma** [comment]
Yes, the dependencies clearly state svg.js 2.x as dependency and not 3.
However we are working on it...

**Comment by alixander** [comment]
Bump. I'm currently using the draggable, filter, resize, and select plugins, and this is the only one that's preventing me from upgrading to 3. Would be eternally grateful for an upgrade on this

**Comment by vCloudSail** [comment]
@alixander 
You can refer to [https://github.com/svgdotjs/svg.js/issues/1146](https://github.com/svgdotjs/svg.js/issues/1146)

**Comment by YTX0412** [comment]
@vCloudSail 
 请问您是怎么解决resize不兼容svg3的？ 我使用您写的那个set函数，解决其他几个插件问题 但是resize插件报错el.doc is not a function

**Comment by Fuzzyma** [comment]
svg.select.js and svg.resize.js are now available under the https://github.com/svgdotjs namespace.
You can install them with `npm i @svgdotjs/svg.select.js @svgdotjs/svg.resize.js`

**Comment by Fuzzyma** [comment]
This was the old mess of mixing the v2 builds with svg.js v3. The plugin has lived under the @svgdotjs scope with proper peer deps for ages now, so I'm closing this.
